src/quo2/components/selectors/filter/__tests__/component_spec.cljs

```diff
@@ -0,0 +1,16 @@
+(ns quo2.components.selectors.filter.--tests--.component-spec
```

Comment:
hmm, the other tests files are "*.<component-name>-component-spec" can we keep it consistent?

In fact since we are putting each component into it's own folder we can probably just have the name space as `quo2.components.selectors.filter.component-spec` for example

Reply:
> In fact since we are putting each component into it's own folder we can probably just have the name space as quo2.components.selectors.filter.component-spec for example

Yes, that's exactly what I wanted to do in this PR, but didn't want to change the conventions too much, but since you mentioned, I can totally make this change in this PR.

Reply:
I can further update the guidelines to mention in cases where the component is isolated in it's own directory, the `__tests__` directory is not necessary and we should use the simpler convention to name the file as `component_spec.cljs` in the same directory of the source component.

Reply:
Yes, the guidelines coming very soon! 😓

Reply:
FYI: Just updated the guidelines.